python/lsst/ts/xml/data/sal_interfaces/Electrometer/Electrometer_Events.xml

```diff
@@ -307,4 +307,65 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>Electrometer</Subsystem>
+    <EFDB_Topic>Electrometer_logevent_fileSummary</EFDB_Topic>
+    <Description>The LFA file of electrometer data is written.</Description>
+    <item>
+      <EFDB_Name>meanIntensity</EFDB_Name>
+      <Description>The mean intensity in the LFA file.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>medianIntensity</EFDB_Name>
+      <Description>The median intensity in the LFA file.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>stdIntensity</EFDB_Name>
+      <Description>The standard deviation of intensity in the LFA file.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>unit</EFDB_Name>
+      <Description>The unit of the LFA file data.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>scanDuration</EFDB_Name>
+      <Description>The duration of the electrometer scan.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>datapoints</EFDB_Name>
+      <Description>The number of datapoints in the LFA file.</Description>
+      <IDL_Type>int</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>filename</EFDB_Name>
+      <Description>The name of the LFA file.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>The TAI time that the data taking began.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
```

Comment:
These two events seem redundant in light of the fits file information as well as the LargeFileArchive available event metadata.